atlasdb-cassandra-integration-tests/src/test/java/com/palantir/atlasdb/keyvalue/cassandra/AbstractCassandraKeyValueServiceIntegrationTest.java

```diff
@@ -255,8 +255,11 @@ private static String getInternalTestTableName() {
     @Test
     @SuppressWarnings("CompileTimeConstant")
     public void shouldNotErrorForTimestampTableWhenCreatingCassandraKvs() {
-        verify(logger, never())
-                .error(startsWith("Found a table {} that did not have persisted"), ArgumentMatchers.<Arg<?>>any());
+        ArgumentCaptor<Arg<String>> argCaptor = ArgumentCaptor.forClass(Arg.class);
+        // A bit strange, but the purpose is to feed the table references to the argument captor.
+        verify(logger, atLeast(0))
+                .error(startsWith("Found a table {} that did not have persisted"), argCaptor.capture());
+        assertThat(argCaptor.getAllValues()).noneMatch(arg -> arg.getValue().contains("timestamp"));
```

Comment:
Can we try this instead? I think it is clearer:

```
        verify(logger, atLeast(0))
                .error(startsWith("Found a table {} that did not have persisted"), assertArg((Arg<String> arg) ->
                        assertThat(arg.getValue()).doesNotContain("timestamp")));
```

Reply:
Thanks - I think it's a good call to use `assertArg(...)`, never heard of that before. That said, I think this as written is a bit different: the [docs](https://javadoc.io/doc/org.mockito/mockito-core/latest/org/mockito/ArgumentMatchers.html) say that "matching is considered successful when the consumer given by parameter does not throw an exception." So I think this verifies that the logger, at least 0 times, is called with a string not containing "timestamp", which would always pass.

I _think_ what we want is `verify(logger, never()).error(startsWith(...), assertArg(arg -> assertThat(arg).contains("timestamp"));` actually. I'll test this

Reply:
oops, never mind - I'm wrong. Thanks for the suggestion again!